### PR TITLE
mflog changes for supporting AWS Lambda

### PR DESCRIPTION
**Pull request by savingoyal**
Ready for review - general housekeeping for #463

### CODE REVIEW

File: metaflow/mflog/redirect_streams.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        else:
+            # Python 3
+            for line in src:
+                decorated = decorate(SOURCE, line)
+                f.write(decorated)
+                dest_stream.buffer.write(line)
+
+
+if __name__ == "__main__":
+    SOURCE = sys.argv[1].encode("ascii")
```
Comment (romain-intel) [inline metaflow/mflog/redirect_streams.py:32]: why ascii and not the more general utf8?

**Comment by savingoyal** [comment]
@oavdeev this PR should unblock #463.

File: metaflow/mflog/__init__.py [hunk 2]
```diff
@@ -43,13 +48,13 @@
 def bash_capture_logs(bash_expr, var_transform=None):
```
Comment (oavdeev) [inline metaflow/mflog/__init__.py:48]: I think it is worth renaming at least the argument, as it cannot be an arbitrary bash expression anymore. Also.. I'm pretty sure there is a bug here (and in the original PR) related to this, in cases when `bash_expr` is not a simple cmd + args

Comment (oavdeev) [reply metaflow/mflog/__init__.py:48]: See https://github.com/Netflix/metaflow/pull/805 for a tentative fix

Comment (savingoyal) [reply metaflow/mflog/__init__.py:48]: I can rename the arg. Keeping the function name constant will make it easier for @romain-intel to merge this PR into the converged branch.

**Pull request merged**